README.md

```diff
@@ -32,6 +32,63 @@ bundle out of these asset-feeds and persist this. It will respond with the URL t
 
 This client helps with remotely triggering these steps in the [asset-pipe-build-server][asset-pipe-build-server].
 
+## Optimistic Bundling
+
+A new approach to bundling assets has been added which we are calling optimistic bundling. This feature allows for assets to react to changes in uploaded assets automatically.
```

Comment:
is "a new" needed? When is it not "new" anymore?

Reply:
Sounds good. Will fix